spec/support/factories.cr

```diff
@@ -255,7 +255,8 @@ def run(command, *, env = nil)
   status = Process.run(command, shell: true, env: cmd_env, output: output, error: error || Process::Redirect::Close)
 
   if status.success?
-    output.to_s.gsub("\r\n", "\n")
+    output = output.to_s.gsub("\r\n", "\n")
+    output + error.to_s
```

Comment:
We should do the line feed normalization for error output as well. This can also be pulled out because it's the same in both conditional branches.